src/cclib/io/ccio.py

```diff
@@ -286,7 +287,7 @@ def ccwrite(ccobj, outputtype=None, outputdest=None, terse=False , returnstr=Fal
 
     Inputs:
         ccobj - Either a job (from ccopen) or a data (from job.parse()) object
-        outputtype - The output format (should be one of 'cjson', 'cml', 'xyz')
+        outputtype - The output format (should be one of 'cjson', 'cml', 'xyz', 'molden')
```

Comment:
I don't think you need to enumerate the keys here, just maybe specify that this is expected to be string. Is it?